contribs/gmf/src/directives/partials/print.html

```diff
@@ -0,0 +1,69 @@
+<div class="print-panel">
+  <div ng-show="ctrl.isState('CAPABILITIES_NOT_LOADED') !== true && ctrl.isState('ERROR_ON_GETCAPABILITIES') !== true">
+    <div ng-repeat="customField in ctrl.fields.customs" class="form-group">
+      <input ng-if="customField.type === 'text'" ng-model="customField.value" class="form-control" type="text" placeholder="{{customField.name|translate}}">
+      <textarea ng-if="customField.type === 'textarea'" ng-model="customField.value" class="form-control" row="3" placeholder="{{customField.name|translate}}"></textarea>
+      <p ng-if="customField.type === 'checkbox'"><input ng-model="customField.value" type="checkbox"> <span translate>Debug</span></p>
```

Comment:
Oups !
That's a custom field. I must display the name of the field. Not "Debug".